### DISCUSSION TITLE
Migrate Git Repository to Git Large File Storage (LFS)

### DISCUSSION THREAD

**Discussion by giscus** [notiz-dev/notiz#184] [General]
# Migrate Git Repository to Git Large File Storage (LFS)

How to migrate an existing Git Repository to use Git Large File Storage (LFS).

https://notiz.dev/blog/migrate-git-repo-to-git-lfs

**Comment by nmikros** [comment]
do you need to commit .gitattributes before doing the importing?

**Comment by raybun** [comment]
Crucial step to avoid confusion and shock after doing a `migrate import`:
Remember to run `git lfs checkout` after the migration, or else the working copies of the affected files will not be repopulated with their expected contents. That one really got me when working on a Unity project with LFS enabled. :)
